phpBB/docs/events.md

```diff
@@ -1595,6 +1595,34 @@ overall_header_stylesheets_after
 * Purpose: Add asset calls after stylesheets within the `</head>` tag.
 Note that INCLUDECSS will not work with this event.
 
+posting_attach_body_attach_row_after
+===
+* Locations:
+    + styles/prosilver/template/posting_attach_body.html
+* Since: 3.2.6-RC1
+* Purpose: Add content after attachment row in the file list
+
+posting_attach_body_attach_row_before
+===
+* Locations:
+    + styles/prosilver/template/posting_attach_body.html
+* Since: 3.2.6-RC1
+* Purpose: Add content before attachment row in the file list
+
+posting_attach_body_attach_row_append
```

Comment:
`posting_attach_body_attach_row_append` 
should go right after
`posting_attach_body_attach_row_after`
if I am not mistaken.

Reply:
yeah, I guess so.